spec/integration/orders_spec.rb

```diff
@@ -30,16 +30,14 @@
     order_mass_g = 100_000
     tonne_per_gram = 1_000_000
 
-    expected_price = (average_price_per_tonne_cents_usd.to_f / tonne_per_gram) * order_mass_g
-
     create_order_response = Patch::Order.create_order(mass_g: order_mass_g, project_id: project_id)
 
     order = create_order_response.data
     expect(create_order_response.success).to eq true
     expect(order.id).not_to be_nil
     expect(order.created_at).to be_a_kind_of(Time)
     expect(order.mass_g).to eq(order_mass_g)
-    expect(order.price_cents_usd).to be_between(expected_price - 2, expected_price + 2)
+    expect(order.price_cents_usd).to be_kind_of(Integer)
```

Comment:
This was failing because that project now had two different prices, so the average logic was broken.

I don't think that the SDK tests are the proper place to test the pricing logic (this should be heavily tested in the API. If it shows up here, it's too late). Removing to reduce brittleness.